docs/source/get_started/model_data/model_data.rst

```diff
@@ -2595,6 +2595,84 @@ copy migration files from installed modules to the :code:`.motech` directory. Fi
 directory(if you are using mysql then use :code:`mysql` instead :code:`default`) in the bundle. Each file muse have
 `a proper name <http://flywaydb.org/documentation/migration/sql.html>`_(e.g. :code:`V1__Description.sql`).
 
+####################
+Schema Import/Export
+####################
+
+This section will explain how to import/export MDS Schema. It is fairly easy and can be done with only few steps using
+the MDS module UI.
+
+Please go to the following subsections to learn how to do that
+
+    - `Schema Import`_
+    - `Schema Export`_
+
+Schema Import
+#############
+
+The first thing you will have to do is to enter the Data Services module UI. You can achieve this by clicking the "Data
+Service" on the left panel.
+
+            .. image:: img/data_services_button.png
+                    :scale: 100 %
+                    :alt: MDS UI button
+                    :align: center
+
+From there you will have to advance to the Settings tab and then enter Import tab.
+
+            .. image:: img/import_schema_tab.png
+                    :scale: 100 %
+                    :alt: MDS Import Schema tab
+                    :align: center
+
+Once in the Import tab, you will have to click the Select file button and choose the file you want to import the schema
+from. Keep in mind that the file must be a valid JSON file. Once you have selected the file, the following screen will
```

Comment:
Moreover, it must also contain a valid schema definition. What happens when I select a valid JSON file that doesn't contain the MDS schema? Please document that.